crates/types/src/parameters.rs

```diff
@@ -87,7 +87,8 @@ pub struct RolePositionsParameters {
     pub striker_supporter_minimum_x: f32,
     pub keeper_x_offset: f32,
     pub striker_distance_to_non_free_center_circle: f32,
-    pub striker_set_position: Point2<Field>,
+    pub striker_kickoff_position: Point2<Field>,
+    pub striker_kickoff_orientation: f32,
```

Comment:
I'd suggest to just make it a `Pose2<Field>`. This is exactly the type to encode position and rotation

Reply:
Because of the way the `Orientation2` is deserialized the ```Pose2``` in the ```default.json``` would have look like this in order to ensure correct rotation
```json
"striker_kickoff_pose": {
        "translation": [0.0, 0.3],
        "rotation": [0.0007962742820382118, -0.9999997019767761]
      }
```
Thoughts?

Reply:
I think this is okay for now, as this is another issue. There may even be an issue already opened for deserializing degree values as unit complexs...